man/ostree.repo-config.xml

```diff
@@ -355,6 +355,11 @@ Boston, MA 02111-1307, USA.
         <listitem><para>If set, pulls from this remote will fail with the configured text.  This is intended for OS vendors which have a subscription process to access content.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>custom-backend</varname></term>
+        <listitem><para>If set, pulls from this remote via libostree will fail with an error.  This is intended to be used by higher level software that wants to fetch ostree commits via some other mechanism, while still reusing the core libostree infrastructure.</para></listitem>
```

Comment:
This is worded like the description of a boolean field (maybe it was in a initial code sketch?).
However my understanding is that the value of this option is really a freeform kind/tag which can be used to dispatch/gate further operations to relevant handlers (including decoding the optional URL too).

Reply:
> (maybe it was in a initial code sketch?).

It was, well spotted.   I tweaked the description.